geonode_mapstore_client/client/js/api/geonode/v2/index.js

```diff
@@ -728,7 +728,14 @@ export const getCompactPermissionsByPk = (pk) => {
 };
 
 export const updateCompactPermissionsByPk = (pk, body) => {
-    return axios.put(parseDevHostname(`${endpoints[RESOURCES]}/${pk}/permissions`), 'permissions=' + JSON.stringify(body))
+    return axios({
+        url: parseDevHostname(`${endpoints[RESOURCES]}/${pk}/permissions`),
+        data: JSON.stringify(body),
```

Comment:
Why do we need to stringify the payload if the content is application/json?

Reply:
@DavidQuartz did you have issues without stringifying?
In that case it must be fixed on the BE